### ISSUE TITLE
Error running Geoclimate with BDTOPO V2.2 [Windows]

### ISSUE THREAD

**Issue by franetibe** [orbisgis/geoclimate#582]
I have downloaded the last version of Geoclimate.jar today.

I run Geoclimate with BDTOPO V2.2 on Windows 10 using this configuration file :

```json
{
    "description": "Processing BD Topo v2 data",
    "geoclimatedb": {
        "folder": "C:\\temp",
        "name": "geoclimate_db_test;AUTO_SERVER=TRUE",
        "delete": true
    },
    "input": {"bdtopo_v2":	{
        "folder": {"path": "C:\\mydirectory\\Geoclimate\\BD_TOPO_v2"}
				}
    },
    "output": {
        "folder": "C:\\temp"
    },
    "parameters": {
        "rsu_indicators": {
            "indicatorUse": [
                "LCZ",
                "TEB",
                "URBAN_TYPOLOGY"
            ],
            "svfSimplified": true
        }
    }
}
```

I obtain the following error, both with CLI and Groovy Console methods :

[Thread-5] INFO org.orbisgis.orbisprocess.geoclimate.bdtopo_v2.BDTopo_V2_Utils - The  BD Topo data have been imported
[Thread-5] INFO org.orbisgis.orbisprocess.geoclimate.bdtopo_v2.BDTopo_V2_Utils - Formating the BDTOPO data
[Thread-5] INFO org.orbisgis.orbisprocess.geoclimate.bdtopo_v2.BDTopo_V2_Utils - The BD Topo data have been formated
[Thread-5] INFO org.orbisgis.orbisprocess.geoclimate.bdtopo_v2.BDTopo_V2_Utils - BDTOPO V2 GIS layers formated
[Thread-5] INFO org.orbisgis.orbisprocess.geoclimate.processingchain.ProcessingChain - Start computing the geoindicators...
[Thread-5] INFO org.orbisgis.orbisprocess.geoclimate.processingchain.ProcessingChain - Create the units of analysis...
[Thread-5] INFO gui.class org.h2gis.network.functions.GraphCreator - Loading graph into memory...
[Thread-5] INFO gui.class org.h2gis.network.functions.GraphCreator -     0.009 seconds
[Thread-5] INFO gui.class org.h2gis.network.functions.ST_ConnectedComponents - Calculating connected components... 
[Thread-5] INFO gui.class org.h2gis.network.functions.ST_ConnectedComponents -     0.003 seconds
[Thread-5] INFO gui.class org.h2gis.network.functions.ST_ConnectedComponents - Storing node connected components... 
[Thread-5] INFO gui.class org.h2gis.network.functions.ST_ConnectedComponents -     0.017 seconds
[Thread-5] INFO gui.class org.h2gis.network.functions.ST_ConnectedComponents - Storing edge connected components...
[Thread-5] INFO gui.class org.h2gis.network.functions.ST_ConnectedComponents -     0.06 seconds
[Thread-5] INFO org.orbisgis.orbisprocess.geoclimate.processingchain.ProcessingChain - Start computing building indicators...
[Thread-5] ERROR org.orbisgis.orbisdata.processmanager.process.Process - Error while executing the process.
org.h2.jdbc.JdbcSQLNonTransientException: Exception lors de l'appel de la fonction définie par l'utilisateur: "buffer(LINESTRING (663464.4 6362216.9, 663455.2 6362206.8, 663451.4 6362202.4, 663447.3 6362199.2, 663438.9 6362193.1, 663431.3 6362188.5, 663427.4 6362185.7), null, 'endcap=flat'): null"
Exception calling user-defined function: "buffer(LINESTRING (663464.4 6362216.9, 663455.2 6362206.8, 663451.4 6362202.4, 663447.3 6362199.2, 663438.9 6362193.1, 663431.3 6362188.5, 663427.4 6362185.7), null, 'endcap=flat'): null"; SQL statement:

CREATE TABLE road_surf_c8b264fa_939e_4302_8d5d_a245cc9937e0 AS 
                    SELECT ST_BUFFER(the_geom, width::DOUBLE PRECISION/2,'endcap=flat') AS the_geom 
                    FROM ROAD; [90105-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:511)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:435)
	at org.h2.message.DbException.get(DbException.java:189)
	at org.h2.message.DbException.convertInvocation(DbException.java:382)
	at org.h2.engine.FunctionAlias$JavaMethod.getValue(FunctionAlias.java:454)
	at org.h2.expression.function.JavaFunction.getValue(JavaFunction.java:40)
	at org.h2.expression.Alias.getValue(Alias.java:37)
	at org.h2.command.dml.Select$LazyResultQueryFlat.fetchNextRow(Select.java:1781)
	at org.h2.result.LazyResult.hasNext(LazyResult.java:101)
	at org.h2.result.LazyResult.next(LazyResult.java:60)
	at org.h2.command.dml.Select.queryFlat(Select.java:721)
	at org.h2.command.dml.Select.queryWithoutCache(Select.java:827)
	at org.h2.command.dml.Query.queryWithoutCacheLazyCheck(Query.java:191)
	at org.h2.command.dml.Query.query(Query.java:476)
	at org.h2.command.dml.Insert.insertRows(Insert.java:218)
	at org.h2.command.dml.Insert.update(Insert.java:152)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:145)
	at org.h2.command.CommandContainer.update(CommandContainer.java:175)
	at org.h2.command.Command.executeUpdate(Command.java:249)
	at org.h2.command.CommandList.update(CommandList.java:65)
	at org.h2.command.CommandList.executeRemaining(CommandList.java:58)
	at org.h2.command.CommandList.update(CommandList.java:66)
	at org.h2.command.Command.executeUpdate(Command.java:249)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:188)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:149)
	at org.h2.jdbc.JdbcCallableStatement.executeUpdate(JdbcCallableStatement.java:79)
	at groovy.sql.Sql.call(Sql.java:3068)
	at groovy.sql.Sql.call(Sql.java:3014)
	at org.orbisgis.orbisdata.datamanager.jdbc.JdbcDataSource.call(JdbcDataSource.java:1146)
	at org.orbisgis.orbisdata.datamanager.jdbc.JdbcDataSource$call$0.call(Unknown Source)
	at org.orbisgis.orbisprocess.geoclimate.geoindicators.BuildingIndicators$_roadDistance_closure5$_closure18.doCall(BuildingIndicators.groovy:394)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:263)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.Closure.call(Closure.java:412)
	at org.orbisgis.orbisdata.processmanager.process.Process.execute(Process.java:300)
	at org.orbisgis.orbisdata.processmanager.process.Process.call(Process.java:466)
	at sun.reflect.GeneratedMethodAccessor78.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
	at org.orbisgis.orbisdata.processmanager.process.Process.invokeMethod(Process.java:416)
	at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at org.orbisgis.orbisprocess.geoclimate.processingchain.GeoIndicatorsChain$_computeBuildingsIndicators_closure1$_closure8.doCall(GeoIndicatorsChain.groovy:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:263)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.Closure.call(Closure.java:412)
	at org.orbisgis.orbisdata.processmanager.process.Process.execute(Process.java:300)
	at sun.reflect.GeneratedMethodAccessor122.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
	at org.orbisgis.orbisdata.processmanager.process.Process.invokeMethod(Process.java:416)
	at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at org.orbisgis.orbisprocess.geoclimate.processingchain.GeoIndicatorsChain$_computeGeoclimateIndicators_closure6$_closure25.doCall(GeoIndicatorsChain.groovy:1589)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:263)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1149)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.Closure.call(Closure.java:412)
	at org.orbisgis.orbisdata.processmanager.process.Process.execute(Process.java:300)
	at sun.reflect.GeneratedMethodAccessor122.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
	at org.orbisgis.orbisdata.processmanager.process.Process.invokeMethod(Process.java:416)
	at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at org.orbisgis.orbisprocess.geoclimate.processingchain.GeoIndicatorsChain$_computeAllGeoIndicators_closure5$_closure18.doCall(GeoIndicatorsChain.groovy:1486)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:263)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1149)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1149)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1149)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.Closure.call(Closure.java:412)
	at org.orbisgis.orbisdata.processmanager.process.Process.execute(Process.java:300)
	at sun.reflect.GeneratedMethodAccessor122.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
	at org.orbisgis.orbisdata.processmanager.process.Process.invokeMethod(Process.java:416)
	at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at org.orbisgis.orbisprocess.geoclimate.bdtopo_v2.WorkflowBDTopo_V2$_bdtopo_processing_closure8.doCall(WorkflowBDTopo_V2.groovy:1322)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:263)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.Closure.call(Closure.java:412)
	at org.codehaus.groovy.runtime.DefaultGroovyMethods.eachWithIndex(DefaultGroovyMethods.java:2234)
	at org.codehaus.groovy.runtime.DefaultGroovyMethods.eachWithIndex(DefaultGroovyMethods.java:2214)
	at org.codehaus.groovy.runtime.DefaultGroovyMethods.eachWithIndex(DefaultGroovyMethods.java:2264)
	at org.codehaus.groovy.runtime.dgm$213.invoke(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite$PojoMetaMethodSiteNoUnwrapNoCoerce.invoke(PojoMetaMethodSite.java:247)
	at org.codehaus.groovy.runtime.callsite.PojoMetaMethodSite.call(PojoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at org.orbisgis.orbisprocess.geoclimate.bdtopo_v2.WorkflowBDTopo_V2.bdtopo_processing(WorkflowBDTopo_V2.groovy:1272)
	at org.orbisgis.orbisprocess.geoclimate.bdtopo_v2.WorkflowBDTopo_V2.bdtopo_processing(WorkflowBDTopo_V2.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:351)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:61)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)
	at org.orbisgis.orbisprocess.geoclimate.bdtopo_v2.WorkflowBDTopo_V2$_workflow_closure1$_closure20.doCall(WorkflowBDTopo_V2.groovy:325)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:263)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.Closure.call(Closure.java:412)
	at org.orbisgis.orbisdata.processmanager.process.Process.execute(Process.java:300)
	at sun.reflect.GeneratedMethodAccessor122.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
	at org.orbisgis.orbisdata.processmanager.process.Process.invokeMethod(Process.java:416)
	at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at ConsoleScript1.run(ConsoleScript1:7)
	at groovy.lang.GroovyShell.runScriptOrMainOrTestOrRunnable(GroovyShell.java:254)
	at groovy.lang.GroovyShell.run(GroovyShell.java:360)
	at groovy.lang.GroovyShell.run(GroovyShell.java:339)
	at groovy.lang.GroovyShell.run(GroovyShell.java:166)
	at groovy.lang.GroovyShell$run$0.call(Unknown Source)
	at groovy.console.ui.Console$_runScriptImpl_closure22.doCall(Console.groovy:1302)
	at groovy.console.ui.Console$_runScriptImpl_closure22.doCall(Console.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:263)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.Closure.call(Closure.java:412)
	at groovy.lang.Closure.call(Closure.java:406)
	at groovy.lang.Closure.run(Closure.java:493)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException
	at org.h2gis.functions.spatial.buffer.ST_Buffer.buffer(ST_Buffer.java:125)
	at sun.reflect.GeneratedMethodAccessor140.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.h2.engine.FunctionAlias$JavaMethod.getValue(FunctionAlias.java:441)
	... 201 more
[Thread-5] INFO org.orbisgis.orbisprocess.geoclimate.processingchain.ProcessingChain - Cannot compute the closest minimum distance to a road at 100 meters.
[Thread-5] ERROR org.orbisgis.orbisprocess.geoclimate.processingchain.ProcessingChain - Cannot compute the building indicators
[Thread-5] ERROR org.orbisgis.orbisprocess.geoclimate.processingchain.ProcessingChain - Cannot build the geoindicators
[Thread-5] ERROR org.orbisgis.orbisprocess.geoclimate.bdtopo_v2.BDTopo_V2_Utils - Cannot build the geoindicators for the zone 12174

Does anyone has an idea ?

Thanks !

**Comment by j3r3m1** [comment]
Thank you for reporting. It seems the road table has null as width, it means that the initial formatting done by the BDTopo does not work perfect since some roads have not been initialized.

@ebocher any recent changes in the BDTOPO that would lead to this behavior ?

@franetibe What is the insee code of the area you are running ?

**Comment by ebocher** [comment]
Nope

**Comment by franetibe** [comment]
@j3r3m1 I am using the test sample presented [here](https://github.com/orbisgis/geoclimate/tree/master/bdtopo_v2/src/test/resources/org/orbisgis/orbisprocess/geoclimate/bdtopo_v2/sample_12174)

**Comment by ebocher** [comment]
It has been updated to use only the COMMUNE file. So the IRIS-GE has been removed.

**Comment by franetibe** [comment]
For this test, iris_ge files have been replaced by commune files.

**Comment by franetibe** [comment]
Here are the results of the geoclimate db test

[geoclimate_db_test.zip](https://github.com/orbisgis/geoclimate/files/6592233/geoclimate_db_test.zip)

**Comment by GAD3L** [comment]
I have exactly the same error... Did you find a solution ? 
Thanks

**Comment by ebocher** [comment]
Is this issue still valid ?

**Comment by j3r3m1** [comment]
Yes I have a try on a windows machine the problem is still there.

**Comment by j3r3m1** [comment]
Solved with version 0.0.1
